source/deployment-options/docker/upgrading-wazuh-docker.rst

```diff
@@ -94,24 +94,48 @@ Single-node stack
+   **Optional**: Update old paths with the new ones based on the version you are upgrading from.
 
-   **Wazuh dashboard**
+   .. tabs::
 
-   #. Edit the ``single-node/config/wazuh_dashboard/opensearch_dashboards.yml`` file and replace ``/usr/share/wazuh-dashboard/config/certs/`` with ``/usr/share/wazuh-dashboard/certs/``.
-   #. Edit the ``single-node/docker-compose.yml`` file and replace ``/usr/share/wazuh-dashboard/config/certs/ with /usr/share/wazuh-dashboard/certs/``.
+      .. group-tab:: Upgrading from 4.3-4.12
 
-   **Wazuh indexer**
+         **Wazuh dashboard**
 
-   #. Edit the ``single-node/config/wazuh_indexer/wazuh.indexer.yml`` file and do the following replacements.
+         #. Edit the ``single-node/config/wazuh_dashboard/opensearch_dashboards.yml`` file and replace ``/usr/share/wazuh-dashboard/config/certs/`` with ``/usr/share/wazuh-dashboard/certs/``.
+         #. Edit the ``single-node/docker-compose.yml`` file and replace ``/usr/share/wazuh-dashboard/config/certs/ with /usr/share/wazuh-dashboard/certs/``.
 
-      -  Replace ``/usr/share/wazuh-indexer/config/certs/`` with ``/usr/share/wazuh-indexer/certs/``.
-      -  Replace ``${OPENSEARCH_PATH_CONF}/certs/`` with ``/usr/share/wazuh-indexer/certs/``.
-   #. Edit the ``single-node/docker-compose.yml`` file and do the following replacements.
+         **Wazuh indexer**
 
-      -  Replace ``/usr/share/wazuh-indexer/config/certs/`` with ``/usr/share/wazuh-indexer/certs/``.
-      -  Replace ``/usr/share/wazuh-indexer/config/opensearch.yml`` with ``/usr/share/wazuh-indexer/opensearch.yml``.
-      -  Replace ``/usr/share/wazuh-indexer/plugins/opensearch-security/securityconfig/`` with ``/usr/share/wazuh-indexer/opensearch-security/``.
+         #. Edit the ``single-node/config/wazuh_indexer/wazuh.indexer.yml`` file and do the following replacements.
+
+            -  Replace ``/usr/share/wazuh-indexer/config/certs/`` with ``/usr/share/wazuh-indexer/certs/``.
+            -  Replace ``${OPENSEARCH_PATH_CONF}/certs/`` with ``/usr/share/wazuh-indexer/certs/``.
+         #. Edit the ``single-node/docker-compose.yml`` file and do the following replacements.
+
+            -  Replace ``/usr/share/wazuh-indexer/config/certs/`` with ``/usr/share/wazuh-indexer/certs/``.
+            -  Replace ``/usr/share/wazuh-indexer/config/opensearch.yml`` with ``/usr/share/wazuh-indexer/opensearch.yml``.
+            -  Replace ``/usr/share/wazuh-indexer/plugins/opensearch-security/securityconfig/`` with ``/usr/share/wazuh-indexer/opensearch-security/``.
+
+      .. group-tab:: Upgrading from 4.13
+
+         **Wazuh dashboard**
+
+         #. Edit the ``single-node/config/wazuh_dashboard/opensearch_dashboards.yml`` file and replace ``/usr/share/wazuh-dashboard/config/certs/`` with ``/usr/share/wazuh-dashboard/certs/``.
+         #. Edit the ``single-node/docker-compose.yml`` file and replace ``/usr/share/wazuh-dashboard/config/certs/ with /usr/share/wazuh-dashboard/certs/``.
```

Comment:
We could take this opportunity and use the following format for consistency wth the rest of the steps and an easy visualization of modifications to perform:

```
...
 #. Edit ``single-node/config/wazuh_dashboard/opensearch_dashboards.yml`` and do the following replacements.
            -  Replace ``/usr/share/wazuh-dashboard/config/certs/`` with ``/usr/share/wazuh-dashboard/certs/``.
 #. Edit ``single-node/docker-compose.yml`` and do the following replacements.
            -  Replace ``/usr/share/wazuh-dashboard/config/certs/`` with ``/usr/share/wazuh-dashboard/certs/``.
...
```